vortex-dtype/src/ptype.rs

```diff
@@ -71,6 +71,18 @@ pub trait NativePType:
     /// Compare another instance of this type to `self`, providing a total ordering
     fn total_compare(self, other: Self) -> Ordering;
 
+    /// Addition, safe on uninitialized data.
```

Comment:
I don't really know what this function name means, and the comment doesn't really help?